src/Workspaces/CoreTest/Execution/SnapshotSerializationTests.cs

```diff
@@ -441,6 +441,33 @@ public async Task RoundTrip_Analyzer_Serailization_Desktop_Test()
             }
         }
 
+        [Fact]
+        public async Task ShadowCopied_Analyzer_Serailization_Desktop_Test()
+        {
+            using (var tempRoot = new TempRoot())
+            {
+                var hostServices = MefHostServices.Create(
+                MefHostServices.DefaultAssemblies.Add(typeof(Host.TemporaryStorageServiceFactory.TemporaryStorageService).Assembly));
+
+                var workspace = new AdhocWorkspace(hostServices);
```

Comment:
❕ need to dispose of workspace before end of test

Reply:
AdhocWorkspace doesn't do anything on Dispose.